pkg/queue/prometheus_stats_reporter_test.go

```diff
@@ -160,25 +165,24 @@ func TestProtobufStatsReporterReport(t *testing.T) {
 				RequestCount:              test.reqCount,
 				ProxiedRequestCount:       test.proxiedReqCount,
 			})
-			checkData(t, requestsPerSecondGV, test.expectedReqCount)
-			checkData(t, averageConcurrentRequestsGV, test.expectedConcurrency)
-			checkData(t, proxiedRequestsPerSecondGV, test.expectedProxiedRequestCount)
-			checkData(t, averageProxiedConcurrentRequestsGV, test.expectedProxiedConcurrency)
-
-			if got := getData(t, processUptimeGV); got < 5.0 || got > 6.0 {
-				t.Errorf("Got %v for process uptime, wanted 5.0 <= x < 6.0", got)
+			want := metrics.Stat{
+				RequestCount:                     test.expectedReqCount,
+				AverageConcurrentRequests:        test.expectedConcurrency,
+				ProxiedRequestCount:              test.expectedProxiedRequestCount,
+				AverageProxiedConcurrentRequests: test.expectedProxiedConcurrency,
 			}
+			got := metrics.Stat{
+				RequestCount:                     getData(t, requestsPerSecondGV),
+				AverageConcurrentRequests:        getData(t, averageConcurrentRequestsGV),
+				ProxiedRequestCount:              getData(t, proxiedRequestsPerSecondGV),
+				AverageProxiedConcurrentRequests: getData(t, averageProxiedConcurrentRequestsGV),
+				ProcessUptime:                    getData(t, processUptimeGV),
+			}
+			cmpStatData(t, want, got)
```

Comment:
Golang generally errs to less assertion functions in tests. As this is only "masking" 3 lines of code, and the ignorance bit isn't even really needed in this test... should we inline the compare logic?

Reply:
Yeah ok.